src/wallet/rpc/transactions.cpp

```diff
@@ -444,7 +444,10 @@ RPCHelpMan listtransactions()
     return RPCHelpMan{
         "listtransactions",
         "If a label name is provided, this will return only incoming transactions paying to addresses with the specified label.\n"
-                "\nReturns up to 'count' most recent transactions skipping the first 'from' transactions.\n",
+                "Returns up to 'count' most recent transactions ordered from oldest to newest while skipping the first 'from' transactions.\n"
+                "Every item in the response array is per output level, i.e., a transaction can have multiple items in the array.\n"
```

Comment:
"output" is kind of a low-level detail, while this is a high-level RPC. It's better to think of entries as logical/financial transactions, as opposed to blockchain transactions.

Reply:
I have not used the term "logical/financial" to describe these transactions yet but I can add it if people believe that would be a valuable addition.